### ISSUE TITLE
Imported "classes" when view in object model are not properly sluggified

### ISSUE THREAD

**Issue by karpana** [Lavaeolous/PF1-StatBlock-Converter-Module#482]
Hi There,

I've been working with the folks on the Foundry VTT discord channel about accessing the 'classes' portion of an imported NPCs statblock, and we've determined that two creature types (Magical Beast and Monstrous Humanoid), when stored in the object model are not being properly sluggified.

here's a link to the conversation in Discord between `Karpana` and `Ethaks`
https://discordapp.com/channels/170995199584108546/711278873047859201/884132898603671573

here's an example of the code that seems to work (for aberration)
```js
let tokenId=args[0][0].current.tokenId    //Note: args is coming from Hook Macro passing the current combattant from the Combat Tracker

let canvasToken = canvas.tokens.get(tokenId)  //locates the token on the canvas for the specified tokenId
let tokenActor = canvasToken.actor            //gets reference to the tokens actor information

// start to spit out to the console, the classes name for the major creature types...
if (tokenActor.data.data.classes.aberration)         {console.log (tokenActor.data.data.classes.aberration.name)}
```

However, Magical Beast and Monstrous Humanoid are not being properly sluggified and thus need to be accessed in this manner.

```js
let tokenId=args[0][0].current.tokenId    //Note: args is coming from Hook Macro passing the current combattant from the Combat Tracker

let canvasToken = canvas.tokens.get(tokenId)  //locates the token on the canvas for the specified tokenId
let tokenActor = canvasToken.actor            //gets reference to the tokens actor information

// start to spit out to the console, the classes name for the major creature types...
if (tokenActor.data.data.classes["magical beast"])         {console.log (tokenActor.data.data.classes["magical beast"].name)}
if (tokenActor.data.data.classes["monstrous humanoid"])         {console.log (tokenActor.data.data.classes["monstrous humanoid"].name)}
```

Ideally, these would be stored so they can be accessed in this manner

```js
let tokenId=args[0][0].current.tokenId    //Note: args is coming from Hook Macro passing the current combattant from the Combat Tracker

let canvasToken = canvas.tokens.get(tokenId)  //locates the token on the canvas for the specified tokenId
let tokenActor = canvasToken.actor            //gets reference to the tokens actor information

// start to spit out to the console, the classes name for the major creature types...
if (tokenActor.data.data.classes.magicalBeast)         {console.log (tokenActor.data.data.classes.magicalBeast.name)}
if (tokenActor.data.data.classes.monstrousHumanoid)         {console.log (tokenActor.data.data.classes.monstrousHumanoid.name)}
```

**Comment by Ethaks** [comment]
To add to this: This affects Racial HD, and the faulty tag creation happens [here](https://github.com/Lavaeolous/PF1-StatBlock-Converter-Module/blob/0f34bb57f3b8f5ed67a81f011b591a700e6eba2e/statblock-converter/scripts/sbcParsers.js#L686-L687).
The system itself uses a `createTag` function to slugify names, and exposes it as part of its API via `game.pf1.utils.createTag`.

**Comment by Lavaeolous** [comment]
Fixed in v3.3.3
